app/services/subjects/financial_incentive_creator_service.rb

```diff
@@ -8,6 +8,25 @@ def initialize(year:, subject: Subject, financial_incentive: FinancialIncentive)
 
     def subject_and_financial_incentives
       subject_and_financial_incentives = {
+        2023 => {
+          %w[Mathematics Physics Chemistry Computing] => {
+            bursary_amount: "27000",
+            scholarship: "29000",
+          },
+          ["French", "German", "Italian", "Japanese", "Mandarin", "Modern languages (other)", "Russian", "Spanish"] => {
+            bursary_amount: "25000",
+            scholarship: "27000",
```

Comment:
How about these?

"Latin", "Ancient Greek", "Ancient Hebrew"

Reply:
If they count as part of that set I'll add them in!

Reply:
done